### PR TITLE
Feature/eui 1253 account info fz

### PR DESCRIPTION
**Pull request by fazanki**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: api/pbaAccounts/index.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    }
+    res.status(500).send(errReport)
+  }
+  const accountNames = req.query.accountNames.split(',')
+  const accounts = []
+  const accountPromises = new Array<AxiosPromise<any>>()
+  accountNames.forEach((accountName: string) => accountPromises.push(getAccount(accountName)))
+
+  try {
+    console.log(accounts)
```
Comment (udaydenduluri33) [inline api/pbaAccounts/index.ts]: Please remove console.log

File: api/pbaAccounts/index.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    await Promise.all(accountPromises).catch(err => err).then(allAccounts => {
+      allAccounts.forEach(account => {
+        const data = account.data === 'Account not found' ? {account_name: 'account not found'} : account.data
```
Comment (udaydenduluri33) [inline api/pbaAccounts/index.ts]: I remember we get a 404 error if account name could not be found. Not sure if account.data does return if Account not found.

File: src/org-manager/containers/organisation-details/organisation-details.component.ts [hunk 3]
```diff
@@ -20,14 +20,28 @@ export class OrganisationDetailsComponent implements OnInit {
...
-   this.orgs$ = this.store.pipe(select(fromStore.getActiveAndPending));
+    this.orgs$ = this.store.pipe(select(fromStore.getActiveAndPending));
+    this.orgs$.pipe(
+        filter(value => value !== undefined),
+        take(1)
+    ).subscribe(({organisationId, pbaNumber, isAccLoaded}) => {
+      if (!isAccLoaded && pbaNumber.length) {
+        this.store.dispatch(new fromOrganisation.LoadPbaAccountsDetails({
+              orgId: organisationId,
+              pbas: pbaNumber.toString()
```
Comment (udaydenduluri33) [inline src/org-manager/containers/organisation-details/organisation-details.component.ts]: pbaNumber is already string. Do we need toString() method?

File: api/pbaAccounts/index.ts (hunk 2 above)
Comment (fazanki) [reply api/pbaAccounts/index.ts]: yes it does find account not found and returns 200 for some odd reason so i am sending it directly to FE

File: src/org-manager/containers/organisation-details/organisation-details.component.ts (hunk 3 above)
Comment (fazanki) [reply src/org-manager/containers/organisation-details/organisation-details.component.ts]: pba in the state are stored as array so at this point i am making sting out of it and sending it to effects -> services

File: api/pbaAccounts/index.ts (hunk 2 above)
Comment (udaydenduluri33) [reply api/pbaAccounts/index.ts]: no worries

Comment (udaydenduluri33) [reply api/pbaAccounts/index.ts]: If you have already tested an Account name and it works that ok. Thanks

**Pull request merged**